### PR TITLE
SPARKC-401 WRTIETIME set using the DataFrame API

### PR DESCRIPTION
**Pull request by mkwatson**

### CODE REVIEW

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala [hunk 1]
```diff
@@ -203,6 +212,13 @@ object WriteConf {
 
     val ttlOption = if(ttlSeconds == TTLParam.default) TTLOption.defaultValue else TTLOption.constant(ttlSeconds)
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: Would you mind fixing the formatting here? I know it's not your patch but I see it now and it burns :)

File: spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala [hunk 2]
```diff
@@ -203,6 +212,13 @@ object WriteConf {
...
 
+
+    val timestampMillis = conf.getLong(TimestampParam.name, TimestampParam.default)
+
+    val timestampOption =
```
Comment (RussellSpitzer) [inline spark-cassandra-connector/src/main/scala/com/datastax/spark/connector/writer/WriteConf.scala]: I believe our style on this is if you have to break multiple lines
http://docs.scala-lang.org/style/control-structures.html
```
If (something)
  do this
else
  do that
```

**Comment by RussellSpitzer** [comment]
Small nits on formatting, if all the tests pass i'm good with this :)
Going to invoke jenkins through the magic words

**Comment by RussellSpitzer** [comment]
test this please

**Comment by mkwatson** [comment]
@RussellSpitzer to fix should I create a new commit amend the previous commit?

**Comment by RussellSpitzer** [comment]
+1 Merging

**Pull request merged**